### DISCUSSION TITLE
I think we should break up SIG-Content

### DISCUSSION THREAD

**Discussion by monroegm** [o3de/sig-content#22] [Ideas]
The scope of SIG-Content is too big to allow for efficient operation of the SIG.
- Over 40% of all O3DE issues fall under SIG-Content, the most of any of the 12 SIGs.
- Even after increasing triage frequency from one to two 30-minute sessions per week, the amount of issues needing triage remains stagnant from week to week.
- We could continue increasing triage frequency but that would increase meeting overhead for individual SIG members. Breaking up the SIG would increase triage frequency without increasing individual overhead.

We could break up SIG-Content into 3 SIGs:
- **SIG-Editor:** Core editing frameworks, components and user interfaces
  - Examples: Viewport, Qt components, Reflected Property Editor
- **SIG-Content-Tools:** Content creation tools that have their own editors and/or behave like standalone tools
  - Examples: EmotionFX (Animation Editor), LyShine (UI Editor), Script Canvas, Lua, Project Manager
- **SIG-Content-Systems:** Systems particularly focused on enhancing content-creation workflows
  - Examples: Prefabs, Terrain, Vegetation

**Comment by AMZN-daimini** [comment]
As discussed during the sig-content meeting, we should define the details of how the transition would work.

@monroegm suggested to add all current sig-content members to all new sigs created after the split, and allow them to opt out of the ones that they are not interested in. I second this approach.

**Comment by AMZN-daimini** [comment]
I'm thinking the proposed break up is not super clear and may lead to confusion in triage.
Since we're striving to create no more than 3 sigs out of content, this is my proposal

- **SIG-Editor**: Core editing frameworks, components and user interfaces
    - Examples: Viewport, Qt components, Reflected Property Editor
     
- **SIG-Content-Tools**: Content creation tools
    - Examples: EmotionFX (Animation Editor), Project Manager, Terrain, Vegetation
    
- **SIG-Game-Logic**: Systems related to game logic
    - Examples: Prefabs, LyShine (UI Editor), Script Canvas, Lua
    
Very open to discuss naming and grouping, as there's definitely no right answer and my subdivision mostly comes out of trying to ease the triage process, while matching areas that have a lot of interaction in terms of code (with the exception of Tools that is quite loose :D )

**Comment by GaianHelmers** [comment]
I agree fully with this desire.
We did what we could by introducing the #gems-and-features forums in the discord server to allow for targeted discussion and better aggregation of information around the features. It was essentially made to stymie the onslaught of varying demands and issues all piling on the one chat channel.

I think the only concern is the need for chairs, and in that, the infrastructure of setting up meetings and handling all that governance stuff.
Is there enough going on to require so many simultaneous efforts?
(That being said: Tools could definitely use some dedicated initiatives and community involvement.)

------

Perhaps a first pass fracture between Content and Tooling (includes tools and editor), and then the anticipation that if heavier work starts happening for the Tooling sig, that it would then fracture into Editor and Tooling?

**Reply by GaianHelmers** [reply]
And to be fair, the features threads have been able to really focus users around target features.
We have:
- Open Particle
- Entity Activation Handling
- QT6
- Assimp alternative implementations
- Jolt Physics
- Hot Reloading
- Component Generator GUI

 and others, all underway.
